syft/format/common/spdxhelpers/to_format_model_test.go

```diff
@@ -938,22 +941,25 @@ func Test_otherLicenses(t *testing.T) {
 			expected: nil,
 		},
 		{
-			name:     "other licenses includes original text",
+			name:     "other licenses must include some original text",
 			packages: []pkg.Package{pkg2},
 			expected: []*spdx.OtherLicense{
 				{
 					LicenseIdentifier: "LicenseRef-non-spdx-license",
-					ExtractedText:     "non spdx license",
+					LicenseName:       "non spdx license",
+					ExtractedText:     "UNKNOWN",
 				},
 			},
 		},
 		{
-			name:     "big licenses get hashed",
+			name:     "big licenses get hashed and space is trimmed",
 			packages: []pkg.Package{pkg3},
 			expected: []*spdx.OtherLicense{
 				{
-					LicenseIdentifier: "LicenseRef-e9a1e42833d3e456f147052f4d312101bd171a0798893169fe596ca6b55c049e",
-					ExtractedText:     bigText,
+					LicenseIdentifier: "LicenseRef-3f17782eef51ae86f18fdd6832f5918e2b40f688b52c9adc07ba6ec1024ef408",
+					// Carries through the syft-json license value when we shasum large texts
+					LicenseName:   "LicenseRef-sha256:3f17782eef51ae86f18fdd6832f5918e2b40f688b52c9adc07ba6ec1024ef408",
```

Comment:
is the added `sha256:` intentional? are we detecting this or creating this? its odd that the identifier and name are different.

Reply:
or is this an spdx limitation?

Reply:
This `LicenseRef-sha256:` is what would be in `Value` of a `syft-json` license if we found the content, could not identify the license, and needed to shasum it's contents for a value. We can totally drop it from the spdx model when translating a syft license -> spdx.